charts/gardener-extension-admission-cilium/charts/runtime/templates/deplyoment.yaml

```diff
@@ -107,3 +115,21 @@ spec:
               audience: {{ .Values.global.serviceAccountTokenVolumeProjection.audience }}
               {{- end }}
       {{- end }}
+      {{- if .Values.global.projectedKubeconfig }}
+      - name: kubeconfig
+        projected:
+          defaultMode: 420
```

Comment:
Any particular reason for this `defaultMode`? Above `readOnly: true` indicates that we do not need write permissions at all. Then `defaultMode: 292` or `defaultMode: 0444` would be sufficient. However, I am not sure why access via group/others should even be allowed at all.

Am I missing something?

Reply:
We use this mode all over the place, e.g. https://github.com/gardener/gardener/blob/bd65783c5d149f9d0510b5a5c8b486754560acaf/pkg/utils/gardener/shoot.go#L426. Unfortunately, I can't for what reasons 😄

Reply:
Works for me.